ufs/ccpp/data/MED_typedefs.F90

```diff
@@ -348,8 +358,12 @@ subroutine interstitial_create(interstitial, im)
     interstitial%prslki = clear_val
     allocate(interstitial%wet(im))
     interstitial%wet = .false.
-    allocate(interstitial%use_flake(im))
-    interstitial%use_flake = .false.
+    allocate(interstitial%use_lake_model(im))
+    interstitial%use_lake_model = .false.
```

Comment:
why is this initialized false if use_lake_model is an integer?

Reply:
right, it is equivalent, but is there a reason to choose .false. rather than 0. Isn't it clearer to use 0 in this case?

Reply:
I just fixed that.

Reply:
sorry--I didn't see your commit below. never mind!